pkg/flare/archive.go

```diff
@@ -746,6 +762,33 @@ func zipHTTPCallContent(tempDir, hostname, filename, url string) error {
 	return err
 }
 
+func zipPerformanceProfile(tempDir, hostname string, profile *Profile) error {
+	cpuProfilePath := filepath.Join(tempDir, hostname, cpuProfileName)
+	cpuProfile, err := os.OpenFile(cpuProfilePath, os.O_RDWR|os.O_CREATE, os.ModePerm)
+	if err != nil {
+		return err
+	}
+	defer cpuProfile.Close()
+
+	heapProfilePath := filepath.Join(tempDir, hostname, heapProfileName)
+	heapProfile, err := os.OpenFile(heapProfilePath, os.O_RDWR|os.O_CREATE|os.O_APPEND, os.ModePerm)
+	if err != nil {
+		return err
+	}
+	defer heapProfile.Close()
+
+	_, err = cpuProfile.Write(profile.CPUProfile)
+	if err != nil {
+		return err
+	}
+	_, err = heapProfile.Write(profile.FirstHeapProfile)
+	if err != nil {
+		return err
+	}
+	_, err = heapProfile.Write(profile.SecondHeapProfile)
+	return err
```

Comment:
Does writing both profile to the same file won't break `pprof` tools ? Should we not write them to 2 different files ?

Reply:
Yes. Will change together with the file suffix.